exonum-java-binding-core/src/test/java/com/exonum/binding/proxy/NativeHandleTest.java

```diff
@@ -23,53 +23,53 @@
 
 import org.junit.jupiter.api.Test;
 
-
 class NativeHandleTest {
 
   private NativeHandle nativeHandle;
 
+  private static long HANDLE = 0x11L;
+
+  private static final String HANDLE_STRING_REPRESENTATION =
+      Long.toHexString(HANDLE).toUpperCase();
```

Comment:
"11";

Reply:
> I can't say it is better, it is not the same constant that you reuse in code because it must have the same value. It just happens that tests use the same value, therefore, extracting these constants does not make them clearer, on contrary — it makes them harder to understand, because you can't immediately see that the second is another representation of the first.
- No ?

Reply:
I am not sure I understand what you mean. No to what and why?

Reply:
Trying to make HANDLE_STRING_REPRESENTATION more clear.